Examples/tvOSMultiplePlayersTemplate/tvOSMultiplePlayersTemplate/PlayersCollectionViewController.swift

```diff
@@ -0,0 +1,87 @@
+
+import UIKit
+import OneMobileSDK
+import AVFoundation
+
+class PlayersCollectionViewController: UICollectionViewController {
+    
+    var playerControllers:[SystemPlayerViewController] = []
+    var playersQuantity = 1
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        for _ in 0..<self.playersQuantity {
+            var provider = OneSDK.Provider.default
+            provider.context.extra = ["noAds": true]
+            provider.getSDK()
+                .then { $0.getPlayer(videoID:"593967be9e45105fa1b5939a", autoplay: false) }
+                .dispatch(on: .main)
+                .onComplete(callback: handle)
+        }
+        
+    }
+    
+    func handle(result: Result<One.Player>) {
+        switch result {
+        case .value(let player):
+            
+            let systemPlayerViewController = SystemPlayerViewController()
+            systemPlayerViewController.player = player
+            systemPlayerViewController.view.frame = CGRect(x: 0, y: 0, width: 1800, height: 768)
+            
+            
+            self.playerControllers.append(systemPlayerViewController)
+            
+            let index = self.playerControllers.count - 1
+            
+            _ = player.addObserver({ (props) in
+                if let isPlaybackReady = props.playbackItem?.content.isPlaybackReady {
+                    print("isPlaybackReady = ", isPlaybackReady, "for player # ", index + 1)
+                }
+            })
+
+            let indexInArray = [IndexPath(row: index, section: 0)]
+
+            if let collection = self.collectionView {
+                collection.reloadItems(at: indexInArray)
+            }
+            
+        case .error(let error):
+            let alert = UIAlertController(title: "Error",
+                                          message: "\(error)",
+                preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "OK",
+                                          style: .default,
+                                          handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        }
+    }
+    
+    // MARK: UICollectionViewDataSource
+    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return playersQuantity
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PlayersCollectionCellId", for: indexPath)
+        
+        if let playerCell = cell as? PlayersCollectionViewCell, indexPath.row < playerControllers.count {
+            playerCell.contentView.superview.map {$0.removeFromSuperview() }
+            let playerVC = self.playerControllers[indexPath.row]
+            playerCell.contentView.addSubview(playerVC.view)
```

Comment:
Adding controller like this violates rules described in:
https://developer.apple.com/library/content/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html